simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/StationMessageSenderTest.java

```diff
@@ -231,6 +233,30 @@ void verifyHeartBeat() throws InterruptedException, JsonProcessingException {
         assertThat(actualMessage.getData().get()).isEqualTo(expectedCall);
     }
 
+    @Test
+    void verifyNotifyReport() throws InterruptedException, JsonProcessingException {
+        List<ReportDatum> reportData = singletonList(new ReportDatum());
+        ZonedDateTime now = ZonedDateTime.now();
+
+        NotifyReportRequest request = new NotifyReportRequest()
+                .withTbc(false)
+                .withSeqNo(0)
+                .withReportData(reportData)
+                .withGeneratedAt(now);
+
+        stationMessageSender.sendNotifyReport(null, false, 0, now, reportData);
```

Comment:
tbc can be used as local variable

Reply:
I know, i choose to do it this way, because 

<img width="1398" alt="Screenshot 2019-03-22 at 14 06 49" src="https://user-images.githubusercontent.com/47893389/54824634-cb926b80-4cab-11e9-99c4-64bfe58ae799.png">

Reply:
Well, we have many places where Idea tries to help but that does not mean that there is always a common sense. 

For example if you have 4 times 'false' usages in your method that mean the same thing (like 'tbc') will you use 'false' everywhere or instead you would create a variable with the value.